include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -241,6 +241,15 @@ class CommonOperatorLoader : public ProtobufLoader {
     nodeValueByName_[op.output(0)] = NodeValue(in, 0);
   }
 
+  void loadCast(const OpType &op, ArgumentDictionaryTy &dict) {
+    auto in = getNodeValueOrCreateVariableByName(op.input(0));
+    assert(in.getElementType() == ElemKind::FloatTy &&
+           "Only float to float cast is currently supported.");
+    int to = loadInt(dict["to"]);
+    assert(to == 1 && "Only float to float cast is currently supported.");
```

Comment:
Can we create an enum of the element types with their id values from Caffe2/ONNX? Instead of hardcoding `1` here.

Reply:
Done. This enum already exists in proto, but unfortunately it is different for Caffe2 and ONNX. Moved the code to Caffe2 importer.